modules/ai/assets/js/editor/components/prompt-library-link.js

```diff
@@ -6,7 +6,7 @@ const PromptLibraryLink = ( props ) => {
 	return (
 		<Typography variant="body2" color="text.secondary">
 			{ __( 'For more inspiration, try experimenting with proven prompts from our' ) }{ ' ' }
-			<Link href={ props.libraryLink } className="elementor-clickable">
+			<Link href={ props.libraryLink } className="elementor-clickable" target="_blank" rel="noopener noreferrer">
```

Comment:
> Note: Setting target="_blank" on <a> elements implicitly provides the same rel behavior as setting [rel="noopener"](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener) which does not set window.opener.
```suggestion
			<Link href={ props.libraryLink } className="elementor-clickable" target="_blank">
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#:~:text=Note%3A%20Setting%20target%3D%22_blank%22

Reply:
done